src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,72 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(toFileName, createReport(readFile(fromFileName)));
```

Comment:
When everything is written in one line it's not very readable. Write the results of the methods in variables